apps/user/filters.py

```diff
@@ -1,7 +1,8 @@
 import django_filters
 from django.db import models
-
+from django.db.models.functions import Concat
 from utils.graphene.filters import IDFilter
+from .models import Q
```

Comment:
Let's not do this, directly use Q from `django.db.models` which is already imported.